Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Validation/PIDWithAutotuningAmigoFOTD.mo

```diff
@@ -0,0 +1,126 @@
+  Buildings.Controls.OBC.CDL.Continuous.Derivative derivative1
+    "A derivative block that is used to mimic the first order process 1"
+    annotation (Placement(transformation(extent={{78,26},{58,46}})));
+  Buildings.Controls.OBC.CDL.Continuous.Derivative derivative2
+    "A derivative block that is used to mimic the first order process 2"
+    annotation (Placement(transformation(extent={{80,-60},{60,-40}})));
+
+equation
+  connect(resSig.y, PI.trigger) annotation (Line(points={{-58,70},{-30,70},{-30,
+          40},{-16,40},{-16,48}}, color={255,0,255}));
+  connect(PIWitAutotuning.trigger, PI.trigger) annotation (Line(points={{-16,-32},
+          {-16,-38},{-30,-38},{-30,40},{-16,40},{-16,48}}, color={255,0,255}));
+  connect(PIWitAutotuning.u_s, PI.u_s) annotation (Line(points={{-22,-20},{-48,-20},
+          {-48,60},{-22,60}}, color={0,0,127}));
+  connect(SetPoint.y, PI.u_s) annotation (Line(points={{-58,10},{-48,10},{-48,60},
+          {-22,60}}, color={0,0,127}));
+  connect(PIWitAutotuning.y, uniDel2.u)
+    annotation (Line(points={{2,-20},{8,-20}}, color={0,0,127}));
+  connect(uniDel1.u, PI.y)
+    annotation (Line(points={{8,60},{2,60}}, color={0,0,127}));
+  connect(uniDel1.y, sub1.u1) annotation (Line(points={{32,60},{40,60},{40,86},{
+          58,86}}, color={0,0,127}));
+  connect(k.y, derivative1.k) annotation (Line(points={{158,30},{148,30},{148,44},
+          {80,44}}, color={0,0,127}));
+  connect(derivative1.T, T.y) annotation (Line(points={{80,40},{112,40},{112,-46},
+          {148,-46},{148,-10},{158,-10}}, color={0,0,127}));
+  connect(derivative1.y, sub1.u2) annotation (Line(points={{56,36},{50,36},{50,74},
+          {58,74}}, color={0,0,127}));
+  connect(sub1.y, PI.u_m) annotation (Line(points={{82,80},{88,80},{88,54},{46,54},
+          {46,42},{-10,42},{-10,48}}, color={0,0,127}));
+  connect(sub2.u1, uniDel2.y) annotation (Line(points={{58,-4},{40,-4},{40,-20},
+          {32,-20}}, color={0,0,127}));
+  connect(derivative2.y,sub2. u2) annotation (Line(points={{58,-50},{52,-50},{
+          52,-16},{58,-16}},
+                          color={0,0,127}));
+  connect(sub2.y, PIWitAutotuning.u_m) annotation (Line(points={{82,-10},{88,-10},
+          {88,-26},{46,-26},{46,-38},{-10,-38},{-10,-32}}, color={0,0,127}));
+  connect(derivative2.k, derivative1.k) annotation (Line(points={{82,-42},{92,-42},
+          {92,44},{80,44}}, color={0,0,127}));
+  connect(derivative2.T, T.y) annotation (Line(points={{82,-46},{148,-46},{148,
+          -10},{158,-10}},         color={0,0,127}));
+  connect(derivative1.u, sub1.u1) annotation (Line(points={{80,36},{88,36},{88,20},
+          {40,20},{40,86},{58,86}}, color={0,0,127}));
+  connect(derivative2.u, uniDel2.y) annotation (Line(points={{82,-50},{92,-50},
+          {92,-66},{40,-66},{40,-20},{32,-20}}, color={0,0,127}));
+  annotation (
+    experiment(
+      StopTime=10000,
+      Tolerance=1e-06,
+      __Dymola_Algorithm="Cvode"),
```

Comment:
We should not have the `__Dymola_Algorithm` set here. It should be set in the .mos script.

Reply:
* [ ]  in `Buildings.Controls.OBC.CDL.Continuous.Validation.PID`, it should not have `__Dymola_Algorithm="Cvode"`

I couldn't locate the issue, can you provide more information? 

 * [x]  remove the .mos file `Resources/Scripts/Dymola/Controls/OBC/Utilities/PIDWithAutotuning/Validation/PIDWithAutotuningAmigoFOTD.mos` and `Resources/Scripts/Dymola/Controls/OBC/Utilities/PIDWithAutotuning/Validation/PIWithAutotuningAmigoFOTD.mos`
* [x]  remove `Resources/Scripts/OpenModelica/compareVars/Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Validation.PIWithAutotuningAmigoFOTD.mos`
 * [x] remove `Resources/Scripts/OpenModelica/compareVars/Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Validation.PIDWithAutotuningAmigoFOTD.mos`
 * [x]  in folder `Resources/ReferenceResults/Dymola`, remove the reference results for the model `Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Validation.PIWithAutotuningAmigoFOTD` and `Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Validation.PIDWithAutotuningAmigoFOTD`

Reply:
About the `__Dymola_Algorithm`, please check https://github.com/SenHuang19/modelica-buildings/blob/9fe60db3d72d9603b30e6efa6a50e0104ee1b68b/bin/verifyFiles.py#L27.

Reply:
Thanks a lot! I restored the accidental changes.